projects/demo/src/modules/components/input/examples/4/index.ts

```diff
@@ -7,7 +7,10 @@ import {TUI_DEFAULT_MATCHER, tuiReplayedValueChangesFrom} from '@taiga-ui/cdk';
 import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
 import {map} from 'rxjs/operators';
 
-import {default as avatar} from '!!file-loader!../../../../../assets/images/avatar.jpg';
+const avatar = new URL(
```

Comment:
Seems like it is used quite often, maybe we can move it to some constant?

Reply:
See https://github.com/Tinkoff/taiga-ui/pull/2110#discussion_r919783061